scripts/check-message-compatibility.py

```diff
@@ -151,14 +158,14 @@ def main(repo1: str, repo2: str, verbose: bool = False):
     else:
         if verbose:
             for msg_type in incompatible_types:
-                file1 = os.path.join(repo1, 'msg', f'{msg_type}.msg')
-                file2 = os.path.join(repo2, 'msg', f'{msg_type}.msg')
+                file1 = find_file_in_subfolders(repo1, f'{msg_type}.msg')
```

Comment:
I'd keep the `msg` part to ensure the same file is found as above
```suggestion
                file1 = find_file_in_subfolders(os.path.join(repo1, 'msg'), f'{msg_type}.msg')
```

Reply:
fixed